core/src/main/scala/org/apache/spark/rdd/RDD.scala

```diff
@@ -1089,6 +1097,14 @@ abstract class RDD[T: ClassTag](
    * and one operation for merging two U's, as in scala.TraversableOnce. Both of these functions are
    * allowed to modify and return their first argument instead of creating a new U to avoid memory
    * allocation.
+   *
+   * @param zeroValue the initial value for the accumulated result of each partition for the seqOp
+   *                  operator, and also the initial value for the combine results from different
+   *                  partitions for the conbOp operator.
+   *                  - this will typically be the neutral element.
```

Comment:
There are some typos now, like conbOp instead of combOp. Also there's a dangling phrase in both starting with "-" which is supposed to join the previous sentence? while you're here you can back-tick `Nil`